lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -3201,6 +3204,11 @@ IRGenDebugInfoImpl::emitFunction(const SILDebugScope *DS, llvm::Function *Fn,
       Name = getName(DS->Loc);
   }
 
+  llvm::DISubprogram *ReplaceableType = DBuilder.createTempFunctionFwdDecl(
```

Comment:
nit: as someone unfamiliar with this API, parameter comments could help contextualize the various constants that are being passed through. although i see that isn't universally done in this file.